cmake-proxies/cmake-modules/dependencies/wxwidgets.cmake

```diff
@@ -74,6 +80,8 @@ if( ${_OPT}use_wxwidgets STREQUAL "system" )
 
     set( toolkit "${wxWidgets_LIBRARIES}" )
 
+    message(STATUS "Trying to retrieve GTK version from ${toolkit}")
```

Comment:
There is no guarantee that $toolkit has "gtk2" or "gtk3" in its name, I am not sure that automagic is needed here, but it will be better to default to gtk-3.0, not gtk-2.0